packages/admin/resources/views/components/layouts/base.blade.php

```diff
@@ -13,6 +13,10 @@ class="filament antialiased bg-gray-100 js-focus-visible"
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="csrf-token" content="{{ csrf_token() }}">
 
+        @foreach (\Filament\Facades\Filament::getMeta() as $tag)
+        {!! $tag !!}
```

Comment:
This doesn't feel right... we should probably use a normal `{{ }}` here instead and in the documentation use `new HtmlString` for the tags instead. Reduces the likelihood of injections etc.

Reply:
This should also be indented.